database/postgres.js

```diff
@@ -10,19 +10,10 @@ const config = {
 
 const pool = new Pool(config);
 
-// const getRestaurant = function (id, cb) {
-//   pool.query(`SELECT * FROM restaurants WHERE id=${id}`, (err, data) => {
-//     if (err) {
-//       cb(err);
-//     } else {
-//       cb(null, data.rows[0]);
-//     }
-//   });
-// };
-
-
-const getRestaurant = (id) => (
+const getRestaurant = (id, cb) => (
   pool.query(`SELECT * FROM restaurants WHERE id=${id}`)
+  .then(res => cb(null, res.rows[0]))
```

Comment:
Sort of weird to be using cb with promises.

Reply:
`const getRestaurant = (id) => (
   pool.query(`SELECT * FROM restaurants WHERE id=${id}`)`

in your server.js
`Places.getRestaurant(req.params.id)
.then(result => response.send(result.rows[0]))
.catch(error => response.status(500).send(error));`